docs/fundamentals/syslib-diagnostics/source-generator-overview.md

```diff
@@ -23,7 +23,6 @@ The following table provides an index to the `SYSLIB1XXX` diagnostics in .NET 6
 | [SYSLIB1001][1001] | Logging method names cannot start with `_`                                                                   |
 | [SYSLIB1002][1002] | Don't include log level parameters as templates in the logging message                                       |
 | [SYSLIB1003][1003] | `InvalidLoggingMethodParameterNameTitle`                                                                     |
-| [SYSLIB1004][1004] | Logging class cannot be in a nested type                                                                     |
```

Comment:
@IEvangelist I removed this article because it refers to an error that only occurred in preview versions.